lib/msf/core/rpc/v10/client.rb

```diff
@@ -69,10 +69,10 @@ def call(meth, *args)
     res = @cli.send_recv(req)
     @cli.close
 
-    if res and [200, 401, 403, 500].include?(res.code)
+    if res && [200, 401, 403, 500].include?(res.code)
       resp = MessagePack.unpack(res.body)
 
-      if resp and resp.kind_of?(::Hash) and resp['error'] == true
+      if resp && resp.kind_of?(::Hash) && resp['error']
```

Comment:
There are various places in Pro that return an error key with an empty string.  Since an empty string is truthy, this causes this to raise a `ServerException` in places it use to not.